src/qt_gui/main_window.h

```diff
@@ -39,12 +40,15 @@ class MainWindow : public QMainWindow {
     void InstallDragDropPkg(std::filesystem::path file, int pkgNum, int nPkg);
     void InstallDirectory();
 
+    std::map<Uint32, KeysMapping> getKeysMapping();
+
 private Q_SLOTS:
     void ConfigureGuiFromSettings();
     void SaveWindowState() const;
     void SearchGameTable(const QString& text);
     void RefreshGameTable();
     void HandleResize(QResizeEvent* event);
+    void keyboardConfigurationButtonPressed();
```

Comment:
```suggestion
    void KeyboardConfigurationButtonPressed();
```
Ditto for all other functions to match the case of the rest of the codebase